lab_3/main.py

```diff
@@ -2,8 +2,47 @@
 Lab 3
 Language classification using n-grams
 """
-
+import json
+import math
 from typing import Dict, Tuple
+import re
+
+
+def _split_into_letters(token: str) -> tuple:
```

Comment:
The top-level functions are not usually named like this - the `_` indicates a protected *method*, i.e. it should be a member of a class. 
This does not apply here - use the regular naming.